BrickController2/BrickController2/UI/Services/Dialog/GameControllerEventDialogResult.cs

```diff
@@ -4,15 +4,19 @@ namespace BrickController2.UI.Services.Dialog
 {
     public class GameControllerEventDialogResult
     {
-        public GameControllerEventDialogResult(bool isOk, GameControllerEventType eventType, string eventCode)
+        public const string NoControllerDeviceId = "NoControllerDeviceId";
```

Comment:
I would add a new constructor for that `false` case instead of having such constant. And probably internally add empty string.

Reply:
It's done:
https://github.com/vicocz/brickcontroller2/pull/75/commits/027b7533c5d58d2228882c1e76943e04ec69c9ca